src/lib/mongoBackend/mongoUpdateSubscription.cpp

```diff
@@ -803,15 +802,16 @@ void updateInCache
 
   if (mscInsert == 0)  // 0: Insertion was really made
   {
-    subCacheUpdateStatisticsIncrement();
-
-    if (subCacheP != NULL)
+    subCacheStatisticsIncrementUpdates();
+  // FIXME PR: I understand this block is not needed with std::map based
+  // logic
+  /*  if (subCacheP != NULL)
     {
       LM_T(LmtSubCache, ("Calling subCacheItemRemove"));
       subCacheItemRemove(subCacheP);
     }
+   */
```

Comment:
Look to this part in the very first commit of this PR: https://github.com/telefonicaid/fiware-orion/pull/2790/commits/91ab399c4de7816cf876155efb7fc0666b3b7c83#diff-dfc89b08d3094a4b23a8ac8b936b6d98L712

In fact, I think that not doing this change was crashing CB in some functest.